### PR TITLE
Smashing magazine

### PR DESCRIPTION
**Pull request by imahnama**
## Smashing-Magazine

In this project we:
- Created HTML semantics structure for the different sections of the project.
- Used different shades of grey for different sections.
- Used grid and flex for the CSS positioning
- Added few types of fonts.
- Added different types of typography

**Note To  TSE**

I @Phylis05, had problems with my machine and used a friend's machine. When making commits it reflected that the owner(Katherine95) of the machine made the commits.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+  height: 600px;
+  padding-top: 20px;
+  background-color: #fff;
+}
+
+.latest-posts8 {
+  height: 600px;
+  padding-top: 20px;
+  background-color: #fff;
+}
```
Comment (mosesogwo) [inline style.css]: Styles should be represented by one class since they are the same,

File: style.css [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+  height: 600px;
+  background-color: #fff;
+  margin-right: 10px;
+}
+
+.latest-posts2 {
+  height: 600px;
+  margin-right: 10px;
+  background-color: #fff;
+}
```
Comment (mosesogwo) [inline style.css]: Styles should be merged into a single class

File: style.css [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+  width: 100%;
+  height: 600px;
+  background-color: #52504f;
+}
+
+.seo {
+  width: 100%;
+  height: 600px;
+  background-color: #c4c4c4;
+}
```
Comment (mosesogwo) [inline style.css]: Similar styles in these classes should be combined under a single class and only the varying style should be specified separately for each of these classes.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**HI @Phylis05, @imahnama, good work:clap:. Just a few changes are required :fast_forward:**
1. Center the contents within the navbar
2. Wrap the main section of the page with `<main>` tags
3. Each article should be enclosed in `<article>` tags




4. The guides section above the footer is not represented in your heatmap.

image

5. Attend to the comments on your stylesheet.

_**Kindly make these changes and submit another code review request.**_

**Any question? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

File: style.css (hunk 3 above)
Comment (imahnama) [reply style.css]: Sure.

**Comment by lekegitrepo** [comment]
### This PR is under Review by [TSE] _Temitayo Adeleke_

**Comment by lekegitrepo** [comment]
### Hi Team 👏 👏 
**_Good job so far, there's still one requirement from the previous review that you did not implement._**

### As mentioned by the previous reviewer: "The guides section above the footer is not represented in your heatmap."

image

**_The section(design patterns & checklists) in the above image should be after this section on your page:_**

image

**_and above this:_**

image

**_Add this section to your page and submit for review._**

_**If you have any doubt or question contact me on slack.**_

### Reviewed by: _Temitayo Adeleke_

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
### Status: Require changes

**Comment by imahnama** [comment]
Hello @lekegitrepo Kindly double check as we have already implemented that change(guides section) 
Screenshot 2020-03-06 at 13 24 10

**Comment by lekegitrepo** [comment]
# Good job Team 💯 🥇 
## The project is approved and good luck with the next project.

### Reviewed by: Temitayo Adeleke

-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved.

**Pull request merged**